test/e2e/storage/empty_dir_wrapper.go

```diff
@@ -179,10 +179,10 @@ var _ = utils.SIGDescribe("EmptyDir wrapper volumes", func() {
 	/*
 		Release : v1.13
 		Testname: EmptyDir Wrapper Volume, ConfigMap volumes, no race
-		Description: Slow by design [~180 Seconds].
+		Description: Slow by design [~180 Seconds]. SLOW tag removed as the test takes less than 5 minutes to execute
```

Comment:
Remove the "slow by design" since it doesn't meet the definition of slow.

Maybe this description should indicate what the test does.

Reply:
Thanks for the comment. The changes are made as per the comment/ References for SLOW tag has been removed and the description now reads the purpose of the test.